hs_core/models.py

```diff
@@ -1686,6 +1686,12 @@ def size(self):
     def verbose_name(self):
         return self.get_content_model()._meta.verbose_name
 
+    @property
+    def can_be_published(self):
+        if self.can_be_public_or_discoverable() and self.resource_type != 'ToolResource':
```

Comment:
I think it would be better to override this property in the ToolResource app and not do this check ```self.resource_type != 'ToolResource'``` here.